doc/doxygen/src/ForHosts.dox

```diff
@@ -98,21 +98,32 @@
  *   user with an additional manual configuration mechanism for your
  *   specific API session if you so wish.
  *
- * - Create, persist and configure a @ref Context as appropriate for
- *   all calls to the API.
+ * - Create (and configure) a @ref Context as appropriate for
+ *   all calls to the API. The Context instance should be kept alive
+ *   and re-used for API calls that are related to the same user
+ *   session, or logical set of interactions. For example, one per
+ *   document. Use @fqref{hostApi.Manager.createChildContext}
+ *   "createChildContext" to split off a new Context should you need to
+ *   re-configure it for any specific series of actions, whilst
+ *   maintaining the association.
```

Comment:
[Consider] I wonder if this is too much prose for the simple checklist section. Perhaps consider
```suggestion
 * - Create (and configure) a @ref Context as appropriate for
 *   all calls to the API. Maintain the same context object for
 *   all subsequent calls related to the same logical set of interactions.
 *   (Link to somewhere explaining/exampling context more deeply.)
```

Reply:
Had opted to add a little more here as the short version inevitably seems to confuse people without examples.

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/1120/commits/15b27a40230dd7a8573be1f0185be81f9d7899ba) as an alternative?